config/lab-routes.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {

```

Comment:
You should be carefull to put files in this folder, because every file name has special meaning.

Default is the master config, the other will be have the name of the `NODE_ENV`

Reply:
So I can actually access this with a proper name for NODE_ENV?
I just wanted some config for routes but decided not to put too much inside 'default.js' so I made another file.
Should I make a folder to hide it or move to another folder, 'routes' maybe?

Reply:
In your case it's ok but if you provide for example NODE_ENV=lab_routes
it will load this file as a config.